hack/ci-build.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env nix-shell
 #!nix-shell -i bash ../shell.nix
 
-make dist-existing-images
+make ORG=tinkerbell/hook dist-existing-images
```

Comment:
When I run this, I get `make: *** No rule to make target 'dist-existing-images'.  Stop.`